pkg/gui/status_panel.go

```diff
@@ -102,16 +103,15 @@ func (gui *Gui) askForConfigFile(action func(file string) error) error {
 	case 1:
 		return action(confPaths[0])
 	default:
-		menuItems := make([]*types.MenuItem, len(confPaths))
-		for i, file := range confPaths {
-			i := i
-			menuItems[i] = &types.MenuItem{
-				DisplayString: file,
+		menuItems := slices.Map(confPaths, func(path string) *types.MenuItem {
+			return &types.MenuItem{
+				DisplayString: path,
 				OnPress: func() error {
-					return action(confPaths[i])
+					return action(path)
 				},
 			}
-		}
+		})
```

Comment:
In this case, the use of the index in the original code wasn't necessary and actually obscured what was going on because in one place we use `file` and in another we use `confPaths[i]` despite them referring to the same thing. The same mistake can be made with a map of course but I have a feeling it won't be as common a mistake, because adding the index arg is either harder to do (as in my slices package) or at least comes after the element itself (as in the lo package). Something about having to think about the index, and explicitly opt-out of using it via an underscore, whenever doing a for loop nudges you to use it I think.